walkthrough/meadow_cookiecutter/{{cookiecutter.directory_name}}/{{cookiecutter.short_name}}.py

```diff
@@ -1,15 +1,18 @@
-from pathlib import Path
-
 import pandas as pd
 from owid.catalog import Dataset, Table, TableMeta
 from owid.catalog.utils import underscore_table
 from owid.walden import Catalog as WaldenCatalog
 from structlog import get_logger
 
+from etl.helpers import Names
 from etl.paths import DATA_DIR, REFERENCE_DATASET
 from etl.steps.data.converters import convert_walden_metadata
 
 log = get_logger()
+
+# naming conventions
+N = Names(__file__)
```

Comment:
The same suggestion about having a `shared.py` module with all the boilerplate code applies to meadow. Eventually, when we have all the tools we need in either `etl.helpers` or `datautils` there should be almost nothing in `shared.py`.

Reply:
> Eventually, when we have all the tools we need in either etl.helpers or datautils there should be almost nothing in shared.py.

I like the idea that what we put in `shared.py` is meant to be eventually moved to a shared space (and unit tested etc.). I'm not adding it _yet_, but will think about it before next walkthrough improvements.